src/UI/GameOptionSetupController.java

```diff
@@ -61,14 +63,15 @@ public GameOptionSetupController(GameBuilder aGameBuilder)
 
     if( initialPicksMap.containsKey(GAME_OPTIONS_CONFIG_KEY) )
     {
+      GameOption<?>[] chronologicalOptions = new GameOption<?>[] {fowOption, startingFundsOption, incomeOption, weatherOption, unitSchemeOption, tagsOption, securityOption, unitCapOption};
       String fullConfig = initialPicksMap.get(GAME_OPTIONS_CONFIG_KEY);
-      String[] s = fullConfig.split("\\s+"); // Split (string integers)
-      int si = 0; // Split Index
-      while (s[si].isEmpty()) ++si; // Skip any sneaky empty splits
+      String[] configInts = fullConfig.split("\\s+"); // Split (string integers)
+      int configIndex = 0; // Split Index
+      while (configInts[configIndex].isEmpty()) ++configIndex; // Skip empty splits at the start
```

Comment:
Do empty splits correspond to default values? Why would there be throwaway data in there?

Reply:
I think the first version of the code started the line with a space, so I ended up putting guard in there.
Arguably, this should be more robust since it's reading from the filesystem, but /shrug.